src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml

```diff
@@ -9,6 +9,7 @@ services:
             configuration_manager: @elcodi.manager.configuration
             product_repository: @elcodi.repository.product
             shipping_range_repository: @elcodi.repository.shipping_range
+            enabled_methods: %store.payment_bridge.enabled_methods%
```

Comment:
Hmmm, mi no like this. Please, rethink this element, as you cannot use any Store parameter